python/CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ In development
   to TypeError, or ValueError if the missing data character clashes
   (:user:`hyanwong`, :pr:`426`).
 
+- Convenient fast access to the number of children of a node in a tree with a
```

Comment:
Too much detail here: user's don't care how it's implemented.

Reply:
Shortened